README.md

```diff
@@ -25,14 +25,14 @@ We are moving development of Fractal v2.0 and all first-party add-ons into a 'mo
 #### Running locally
 
 1. Clone this repository
-2. Install dependencies - `npm install`
-3. Bootstrap the packages together using Lerna - `npm run bootstrap`
+2. Install dependencies - `yarn install`
```

Comment:
I actually left this as `npm` rather than `yarn` intentionally - the repo includes yarn as a dependency so that lerna and all npm scripts can use it, but the user doesn't need to have Yarn installed to run the tests or install the dependencies. Obviously they can if they want to but I felt like leaving the instructions referenceing `npm` was the most neutral approach. What do you think?